src/emulate_blue_2.2.5.c

```diff
@@ -60,6 +68,24 @@ int emulate_blue_2_2_5(unsigned long syscall, unsigned long *parameters,
            unsigned int,         len,
            uint8_t *,            out,
            unsigned int,         out_len);
+
+  SYSCALL8(os_perso_derive_node_bip32_seed_key, "(0x%x, 0x%x, %p, %u, %p, %p, %p, %u)",
+           unsigned int,         mode,
+           cx_curve_t,           curve,
+           const unsigned int *, path,
+           unsigned int,         pathLength,
+           unsigned char *,      privateKey,
+           unsigned char *,      chain,
+           unsigned char *,      seed_key,
+           unsigned int,         seed_key_length);
+  
+  SYSCALL6(cx_hmac_sha512, "(%p, %u, %p, %u, %p, %u)",
```

Comment:
The `cx_hmac_sha512` syscall ID is the same for each SDK version (ie. `0x600019cf`) and isn't specific to Blue 2.2.5. It can thus be moved to `src/emulate.c`